modules/category/category.service.js

```diff
@@ -30,22 +35,18 @@ class CategoryService {
   }
 
   async addCategory(data, parentId) {
-    if (data.name.length < 2 || !data.code || (!data.isMain && !parentId)) {
-      throw new Error(WRONG_CATEGORY_DATA);
+    if (data.name.length < 2) {
+      throw new Error(CATEGORY_INPUT_ERROR);
```

Comment:
Please, make this error more meaningful and use `joi-validator` here

Reply:
used